src/components/elements/Map-mapbox/Map.tsx

```diff
@@ -1,38 +1,57 @@
 import "mapbox-gl/dist/mapbox-gl.css";
 
+import _ from "lodash";
 //@ts-ignore
-import MapboxDraw from "@mapbox/mapbox-gl-draw";
 //@ts-ignore
-import StaticMode from "@mapbox/mapbox-gl-draw-static-mode";
-import mapboxgl, { Map as IMap } from "mapbox-gl";
+import mapboxgl from "mapbox-gl";
 //@ts-ignore
```

Comment:
Please investigate if any of these `ts-ignore`s are still needed. In particular, having two of them stacked on lines 4-5 should not be necessary.

Reply:
they weren't necessary. Removed.